tests/plugins/uploadwidget/manual/paste.md

```diff
@@ -0,0 +1,38 @@
+﻿@bender-tags: 4.10.2, bug, clipboard, widget, filetools, 1217
+@bender-ui: collapsed
+@bender-ckeditor-plugins: wysiwygarea, toolbar, uploadwidget, image2, uploadimage, uploadfile
```

Comment:
I wonder if it's ok to skip `uploadwidget` plugin, as it is required by other listed plugins. Or is it better to keep it explicitly required like this. WDYT?

Reply:
Good catch, it doesn't make sense to keep plugin if it's not required. Moreover, this test is missing proper `uploadwidget` tag.